www/~/%username/emails/modify.json.spt

```diff
@@ -46,7 +47,10 @@ if action in ('add-email', 'resend', 'start-verification'):
         raise Response(400)
 
     participant.start_email_verification(address, *packages)
-    msg = _("Check your inbox for a verification link.")
+    if show_address_in_message:
```

Comment:
Good find, @EdOverflow.

> I think it'd be good to add a comment in the simplate that mentions this

Comment in c3677d4943a4be57dd3f4ecb0aee16187f3c30c7, with link to H1.

I've also kicked off disclosure for those two. Seems they fell through the cracks. ☺️